builtin/logical/pki/crl_util.go

```diff
@@ -151,10 +155,16 @@ func buildCRL(b *backend, req *logical.Request) error {
 			return errutil.InternalError{Err: fmt.Sprintf("Unable to parse stored revoked certificate with serial %s: %s", serial, err)}
 		}
 
-		revokedCerts = append(revokedCerts, pkix.RevokedCertificate{
+		// NOTE: We have to change this to UTC time because the CRL standard
+		// mandates it but Go will happily encode the CRL without this.
+		newRevCert := pkix.RevokedCertificate{
 			SerialNumber:   revokedCert.SerialNumber,
-			RevocationTime: time.Unix(revInfo.RevocationTime, 0),
-		})
+			RevocationTime: time.Unix(revInfo.RevocationTime, 0).UTC(),
```

Comment:
Will there be a need to distinguish entries that were stored before, in not-UTC?